### DISCUSSION TITLE
Rear driven vehicle in CARLA

### DISCUSSION THREAD

**Discussion by angloth** [carla-simulator/carla#4760] [Q&A]
Hi!

I am working on a project to create a self-driving car that drifts as much as possible. However, I would prefer to use a rear driven car model as it's much easier to initiate and maintain drifts. During some fast investigation it seems like all of the vesicles (including the bicycles) in CARLA apply torque from the motor to all 4 (or 2) wheels, i.e. they are all wheel driven.

Therefore I am wondering if anyone knows a way to emulate a rear-driven (or front-driven) vehicle in CARLA?

**Comment by xZeenos** [comment]
Hi, so i looked around a bit and i think i found an answer! In your vehice bluebrint in the components panel click on "vehicle movement". In the details panel scroll down until you see "mechanical setup". Open this submenu and look for differential setup. change "differential type" to what ever you need. I would recommend a no slip diff but it doesnt really matter then select either front drive or rear drive and you are done.

Im not sure if that is the officiall solution but its the one i found. 

Have a good one :D
